### DISCUSSION TITLE
KS with Azure Load Balancer...

### DISCUSSION THREAD

**Discussion by dellui** [kubesphere/kubesphere#5394] [Q&A]
Hi.
We have deployed KubeSphere on Azure with success and now we would configure Azure LB but we can't find documentation to help us to complete configuration.
Where we can find some informations ?

Thanks.
Ciao.
Gigi

**Comment by binary01010** [comment]
You can find LB with cluster gateway ，LB rename to cluster gateway and soft gateway
